Cython/Compiler/Nodes.py

```diff
@@ -1752,7 +1758,7 @@ class CEnumDefItemNode(StatNode):
 
     child_attrs = ["value"]
 
-    def analyse_declarations(self, env, enum_entry):
+    def analyse_declarations(self, env, enum_entry, incremental_enum_value):
```

Comment:
This should better not be called `analyse_declarations()`. It has a different signature and returns a value. Actually, it analyses the declarations and changes the node state accordingly, but then still returns a value. That's not great. It should do either or, not both.

Reply:
The return value is easy to get rid of by assigning entry to the node so I'll definitely do that.

I'd argue that it already has a non-standard signature though, so I'm not making it too much worse by adding an argument

Reply:
I've removed the return value. I can rename it as `analyse_enum_item_declarations` (or something like that) if you want, but for now, I'm still claiming it already had a non-standard signature.

Reply:
> I'm still claiming it already had a non-standard signature.

Then let's rename it. IMHO, that should have been done when it was first introduced. Maybe just `analyse_enum_declarations`? Or `…_enum_item_…`, as you proposed. I don't mind, just something specific.